### ISSUE TITLE
Fedora 27 - I got error libopencv_imgproc.so 

### ISSUE THREAD

**Issue by catafest** [hybridgroup/gocv#48]
I try to install gocv with Fedora 27  ( https://gocv.io/getting-started/linux/) 
I got errors on make build  :
```
[ 42%] Linking CXX shared library ../../lib/libopencv_imgproc.so
...
make: *** [Makefile:163: all] Error 2
```

**Comment by deadprogram** [comment]
Hi @catafest could you please include the entire build log so we take a closer look? Thanks!

**Comment by deadprogram** [comment]
Hello @catafest could you please provide the full log?

**Comment by deadprogram** [comment]
Any update on this?

**Comment by mpx** [comment]
Fedora 27 has OpenCV 3.2 (opencv-3.2.0-13.fc27.x86_64). The GoCV website indicates 3.3 might be the minimum version. I suspect Fedora 28 might have 3.3, but that is a [few months away](https://fedoraproject.org/wiki/Releases/28/Schedule).

**Comment by deadprogram** [comment]
Thanks for chiming in @mpx that explains it.

Well, GoCV *might* work as long as the proper env vars are set. For anyone who wants to try it out, use the instructions at https://github.com/hybridgroup/gocv#other-linux-installations instead of running `source env.sh`

**Comment by deadprogram** [comment]
I think that the recent contributions from @metal3d to the `env.sh` script might help avoid this problem entirely on Fedora. Please check out the current `dev` branch.

**Comment by deadprogram** [comment]
You could also install OpenCV from source.

Any update on this issue? If not I will close it.

**Comment by deadprogram** [comment]
I'm pretty sure that the `Makefile` in the `dev` branch can now handle Fedora builds without issue.

Anyone able to confirm this?

**Comment by deadprogram** [comment]
This code has now been released as part of v0.12.0 so now closing. Thanks again!
